content/en/docs/setup/platform-setup/azure/index.md

```diff
@@ -4,19 +4,24 @@ description: Instructions to set up an Azure cluster for Istio.
 weight: 10
 skip_seealso: true
 aliases:
-    - /docs/setup/kubernetes/prepare/platform-setup/azure/
-    - /docs/setup/kubernetes/platform-setup/azure/
-keywords: [platform-setup,azure]
+  - /docs/setup/kubernetes/prepare/platform-setup/azure/
+  - /docs/setup/kubernetes/platform-setup/azure/
+keywords: [platform-setup, azure]
 owner: istio/wg-environments-maintainers
 test: no
 ---
 
-This page was last updated September 12, 2019.
-
-{{< boilerplate untested-document >}}
+This page was last updated April 26, 2023
```

Comment:
I'm not sure you need to put the date in here, as we don't have it on the other pages. I believe there was no known testing of this page at one point in time, so the two lines were added so users would know the page may be stale. I think the plan at that time was to simply have the page updated and remove the two lines.

Reply:
Ok I'll remove that. It was there previously, and I didn't know the context of it.